content/2025-09-04_pyvo-na-ep.md

```diff
@@ -0,0 +1,21 @@
+Title: Na Pyvo s EuroPythonem
+Date: 2025-09-04 18:00:00
+Author: Mia Bajić
+Gravatar: miabajic.miabajic@gmail.com
+About: Mia je vývojářka, místopředsedkyně EuroPython Society, a spoluorganizátorka pražského Pyva. 
+
+Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
+
+Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!s).
```

Comment:
> kdo upstream udržuje 200 knihoven

Tohle mi úplně nedává smysl/nesedí. Cc @befeleme

Reply:
já jsem to takto pochopila z popisu přednášky, ale pokud máš návrh, jak to lépe říct, tak klidně ho přidej přes _Add a suggestion_ a akceptuju ho

Reply:
aktualizováno, díky @befeleme!